boards/silabs/dev_kits/xg24_ek2703a/board.cmake

```diff
@@ -3,3 +3,6 @@
 
 board_runner_args(jlink "--device=EFR32MG24BxxxF1536" "--reset-after-load")
 include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
+
+board_runner_args(silabs_commander "--device=EFR32MG24B210F1536IM48")
+include(${ZEPHYR_BASE}/boards/common/silabs_commander.board.cmake)
```

Comment:
move this into the board commit, doesn't make sense to add the board in one commit then edit it right away in the following commit